mathics/builtin/fileformats/htmlformat.py

```diff
@@ -287,6 +294,8 @@ class DataImport(_DataImport):
 
 class FullDataImport(_DataImport):
     """
+    ## <url>:internal native:</url>
```

Comment:
I think we can remove this

Reply:
I want to keep this to complete this in another round. Also because I want to add a pytest that ensures that all the docstrings in Builtin subclasses have the field (at least, commented out).

Reply:
Ok, but please with automation scripts:

* Expect that you are going to get something wrong, so don't shove the entire code base through this. 
* When you encounter something like: Is there a convention for ...  ? and you don't see the pattern, don't just plough ahead. 
* In automation scripts break up changes into batches of _10_ files. (Mathics tells me that 14 > 10, and 26 > 10) .
* Check the _first_ 10,  put in a PR for that and adjust the next run based on problems with the first, e.g. "WMA" vs "WMA link", or breaking long url lines. 

Thanks.